README.md

```diff
@@ -30,30 +30,30 @@
 
 ## What is DevChat?
 
-DevChat is an open-source platform that empowers developers to leverage AI for code generation and documentation. We aim to go beyond simple code auto-completion and limited operations on code snippets. DevChat offers a highly *practical* and *effective* way for developers to interact and collaborate with large language models (LLMs).
+DevChat is an open-source platform that empowers developers to more naturally and effectively integrate AI into code generation and documentation. DevChat aims to go beyond simple auto-completion and limited operations on code snippets. DevChat offers a highly *practical* and *effective* way for developers to interact and collaborate with large language models (LLMs), DevChat is the tool that works for you, no need to change your workflow.
 
 ## Why DevChat?
 
-While there are many AI coding tools available, we developed DevChat based on our practical insights from generating tens of thousands of lines of code. DevChat makes the following distinctive design choices:
+We spent years on code analysis, static-analysis, and DevOps improvements, working with hundreds of companies around the world. The insights we have picked up in this process are all incorporated into DevChat. The techniques, approaches, and decisions that drive success for companies large and small inform every detail of DevChat. You'll find some distinctive design choices:
 
```

Comment:
Hmm... Code analysis makes sense. Most others are not that relevant so it may sound over selling. Maybe we can position us as dev tool builders like creators of Apache DevLake? In addition, the amount of code we generate is anyway better evidence of our experience as this is much more closely related. Tens of thousands lines of code is not a small number -- most developers should know this.